### ISSUE TITLE
replace boot.js for boot.dart

### ISSUE THREAD

**Issue by DartBot** [dart-lang/polymer-dart#156]
<a href="https://github.com/sigmundch"><img src="https://avatars.githubusercontent.com/u/2049220?v=3" align="left" width="96" height="96"hspace="10"></img></a> **Issue by [sigmundch](https://github.com/sigmundch)**
_Originally opened as dart-lang/sdk#18007_

----

this is blocked by a couple issues:

&nbsp;- issue dart-lang/sdk#18001: we need to be able to use spawnDomUri with package imports
&nbsp;- issue dart-lang/sdk#18006: we want to include boot.dart in polymer.html in the same way we include boot.js

**Comment by DartBot** [comment]
<a href="https://github.com/sigmundch"><img src="https://avatars.githubusercontent.com/u/2049220?v=3" align="left" width="48" height="48"hspace="10"></img></a> **Comment by [sigmundch](https://github.com/sigmundch)**

----

Seems that some our plans might be changing here. I deleted the portions of boot.dart that were equivalent to boot.js in the change that reintroduces initPOlymer.

In case it is useful in the future, here is what the code looked like when I deleted it:

&nbsp;&nbsp;&nbsp;&nbsp;main() {
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;var scripts = discoverScripts(document, window.location.href);
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;var sb = new StringBuffer()..write('library bootstrap;\n\n');
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;int count = 0;
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;for (var s in scripts) {
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;sb.writeln("import '${s.importUrl}' as prefix_$count;");
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;count++;
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;}
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;sb.writeln("import 'package:polymer/src/mirror_loader.dart';");
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;sb.write('\nmain() => startPolymerInDevelopment([\n');
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;for (var s in scripts) {
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;sb.writeln("  '${s.importUrl}',");
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;}
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;sb.write(']);\n');
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;var isolateUri = \_asDataUri(sb.toString());
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;spawnDomUri(Uri.parse(isolateUri), [], '');
&nbsp;&nbsp;&nbsp;&nbsp;}

**Comment by DartBot** [comment]
<a href="https://github.com/jmesserly"><img src="https://avatars.githubusercontent.com/u/1081711?v=3" align="left" width="48" height="48"hspace="10"></img></a> **Comment by [jmesserly](https://github.com/jmesserly)**

----

*Removed [Area-Polymer](../labels/Area-Polymer) label.*
*Added [Pkg-Polymer](../labels/Pkg-Polymer), [Area-Pkg](../labels/Area-Pkg) labels.*

**Comment by DartBot** [comment]
<a href="https://github.com/sigmundch"><img src="https://avatars.githubusercontent.com/u/2049220?v=3" align="left" width="48" height="48"hspace="10"></img></a> **Comment by [sigmundch](https://github.com/sigmundch)**

----

Closing this until we set our new plan with the new script semantics.
______
*Added [NotPlanned](../labels/NotPlanned) label.*
